### PR TITLE
Disable ginkgo verbose mode

### LINKED ISSUE #2074: test logging
Our tests are too verbose, it is hard to see what failed.

We don't need to see all the logs for every successful test spec.
But when test fails we should dump all the info that we can.

- remove verbose flag from ginkgo
- add a verbose flag to every odo command that is executed in tests?, Or maybe to just once that are causing test failures and flakes?

### PR DESCRIPTION
**Pull request by amitkrout**
## Was the change discussed in an issue?
fixes #2074

NOTE: Make a practice of adding a verbose flag to every odo command in test file

## How to test changes?

See the test result in the terminal locally

### CODE REVIEW

**Comment by girishramnani** [comment]
we should allow an env variable to enable this too @amitkrout

**Comment by amitkrout** [comment]
> we should allow an env variable to enable this too @amitkrout

Done.

File: scripts/openshiftci-presubmit-integrationtests.sh [hunk 1]
```diff
@@ -7,6 +7,7 @@ set -x
 
 export CI="openshift"
 export TIMEOUT="30m"
+export GINKGO_VERBOSE_MODE="-v"
```
Comment (mik-dass) [inline scripts/openshiftci-presubmit-integrationtests.sh]: I don't see the verbose output in the integration tests running on the openshift cluster like before

File: scripts/openshiftci-presubmit-e2etests.sh [hunk 2]
```diff
@@ -7,6 +7,7 @@ set -x
 
 export CI="openshift"
 export TIMEOUT="30m"
+export GINKGO_VERBOSE_MODE="-v"
```
Comment (mik-dass) [inline scripts/openshiftci-presubmit-e2etests.sh]: https://github.com/openshift/odo/pull/2145/files#r326596049

Comment (amitkrout) [reply scripts/openshiftci-presubmit-e2etests.sh]: here it is - https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_odo/2145/pull-ci-openshift-odo-master-v4.1-integration/257/artifacts/integration/container-logs/test.log

File: scripts/openshiftci-presubmit-integrationtests.sh (hunk 1 above)
Comment (amitkrout) [reply scripts/openshiftci-presubmit-integrationtests.sh]: Locally it is disabled. To enable it locally see the doc changes in this pr.

File: docs/development.adoc [hunk 3]
```diff
@@ -301,6 +301,16 @@ Expect(output).To(ContainSubstring("No file changes detected, skipping build"))
...
 
+*Test variables:*
+
+There are some test environment variable that helps to get more control over the test run and it's results
+
+* TEST_EXEC_NODES: Env variable TEST_EXEC_NODES is used to pass spec execution type (parallel or sequential) for ginkgo tests. To run the specs sequentially use TEST_EXEC_NODES=1, otherwise by default the specs are run in parallel on 2 ginkgo test node. Any TEST_EXEC_NODES value greater than one runs the spec in parallel on the same number of ginkgo test nodes.
+
+* SLOW_SPEC_THRESHOLD: Env variable SLOW_SPEC_THRESHOLD is used for ginkgo tests. After this time (in second), ginkgo marks test as slow. The default value is set to 120s.
+
+* GINKGO_VERBOSE_MODE: Env variable GINKGO_VERBOSE_MODE is used to get control over enabling ginkgo verbose mode against each test target run. By default ginkgo verbosity is not enabled. To enable verbosity export or set env GINKGO_VERBOSE_MODE like "GINKGO_VERBOSE_MODE=-v".
```
Comment (amitkrout) [inline docs/development.adoc]: @mik-dass ^

File: scripts/openshiftci-presubmit-integrationtests.sh (hunk 1 above)
Comment (amitkrout) [reply scripts/openshiftci-presubmit-integrationtests.sh]: @mik-dass https://github.com/openshift/odo/pull/2145/files#r326962153

**Comment by amitkrout** [comment]
```
level=fatal msg="failed to initialize the cluster: Multiple errors are preventing progress:\n* Cluster operator authentication is still updating: missing version information for oauth-openshift\n* Cluster operator console has not yet reported success\n* Cluster operator node-tuning has not yet reported success\n* Cluster operator openshift-samples is still updating\n* Could not update deployment \"openshift-operator-lifecycle-manager/packageserver\" (265 of 349): timed out waiting for the condition"
```
/test v4.1-e2e-scenarios

**Pull request merged**